CONTRIBUTING.md

```diff
@@ -365,6 +366,25 @@ M      | 1000
 
 Using this table, write a sample program which accepts a Roman numeral and outputs it as a decimal.
 
+### Convex Hull
+
+Suppose you have a set of points in the plane. The **convex hull** of this set is the smallest
+convex polygon that contains all the points.
+
+A good way to visualize the problem is this: Imagine each point is a nail sticking out of the plane,
+and you stretch a rubber band around them and let it go. The band will contract and assume a shape
+that encloses the nails. This shape is the convex hull.
+
+![Rubber band visualization](https://upload.wikimedia.org/wikipedia/commons/d/de/ConvexHull.svg)
+
+Note that all vertices of the convex hull are points in the original set. So the convex hull is really
+a subset of points from the original set, and there may be points that lie inside the polygon but are
+not vertices of the convex hull.
+
+There are many algorithms to solve this problem. You may implement any of them. Check this
```

Comment:
Can you include some concrete directions? I think this last paragraph could be changed to indicate what the program should accept (input) and what it should output. The implementation is then up to the developer. 

In other words, how should the input look? Maybe: `[[x1, y1], [x2, y2],..., [xn, yn]]`

How about the output? Maybe the same.

Reply:
That kind of array you are suggesting seems more familiar to languages like JavaScript but for others like Java it would be a little strange. Jeff Erickson suggests taking two arrays X[] and Y[] as the inputs, and return a list of indexes as the output. What do you think?

Reply:
Fair point! I'm actually more comfortable with Java and Python. I was thinking more about the lists conceptually, so a list of points makes more sense. That said, I like the two lists better. And, outputting a list of indices is perfect.

Do you care how the input looks? I think two comma separated strings would be perfect. In fact, I might change my description of #506 to that.

Reply:
You mean like "x1,x2..." and the user makes a split by comma? So the program would receive strings for X and Y and parse them into arrays

Reply:
Well, implementation is up to the user. I'm just trying to come up with the most sensible way to pass the lists to the program. So, yeah: something like `x1, x2, x3` and the user parses that list into some list in their language.

Reply:
Got it. Just making sure the program receives two strings for X and Y. It should still return an array of indices, right?

Reply:
Yup!

Reply:
Why is this discussion marked as resolved? Maybe I'm missing something but the bit about the input format should still be added, right?

Reply:
Yes! Sorry, I resolved it because the discussion is finished, but yeah, the changes still need to be added.

Reply:
Disregard my approval then 🙂 (I can't easily change it in the mobile interface)

Reply:
Yeah, sorry. I've been a little busy but I will complete this later today or tomorrow

Reply:
Hey I just realized that since some of the algorithms for solving this sort the array of points, returning a list of indices might be confusing (because the indices change when the array is sorted). 

So do we enforce that the user keeps track of the "original" indices and use them in the output, or do we just output the coordinates instead?

Reply:
Good point! It probably makes more sense to output the set of coordinates that make up the convex hull but accept two comma separated lists (x and y) as input. That way, input processing is easy (verify input lists are the same length and split by comma). And, output is meaningful.

Reply:
Check the changes I just pushed and let me know!